Cargo.toml

```diff
@@ -44,7 +44,11 @@ sha2 = { version = "0.8", optional = true }
 x25519-dalek = { version = "0.5", optional = true }
 
 # ring crypto proivder
-ring = { version = "0.15", optional = true }
+[dependencies.ring]
+version = "0.16"
```

Comment:
0.16.0 and 0.16.1 don't seem to have have the correct API (the `separate_tag` variant you use) and they're not yanked, so we're going to have to depend on "^0.16.2"